source/variables/typography.css

```diff
@@ -1,13 +1,13 @@
 :root {
 	/*The base font-size for the root html element.  The value represents its pixel equivalent, e.g. 16 = 16px.*/
-    --type-font-size: 16;
+    --typography-font-size: 16;
 
     /*The percentage of the font-size to be used on the root html element.*/
-    --type-font-percentage: calc(calc(var(--type-font-size) / 16) / 100%);
+    --typography-font-percentage: calc(calc(var(--typography-font-size) / 16) / 100%);
     
     /*The base line height for the root html element.*/
-    --type-line-height: 1.5;
+    --typography-line-height: 1.5;
 
     /*The base font-family for the root html element.*/
-    --type-font-family: sans-serif;
+    --typography-font-family: sans-serif;
 }
```

Comment:
Can we change these names to `--type-` and renamed this to type.css? just from an ergonomic perspective `--type` is a lot easier to type of than `--typography`.